conditionals.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Conditionals</title>
+  <meta charset="utf-8">
+  <script>
+    var year = prompt("¿how old are you?");
+    if (year>18){
+      alert("welcome a our page");
+    }
```

Comment:
Add an else statement in case they have less than 18 years. With a text: 

> You can't login if you are underaged.

Reply:
Done!